### ISSUE TITLE
Local run command no longer works

### ISSUE THREAD

**Issue by sverhoeven** [tortellini-tools/action#123]
https://github.com/tortellini-tools/action/blob/main/README.dev.md#on-the-current-repository describes how to run check locally however this gives error:

```shell
ode dist/index.js
::error::no such file or directory: 
```

Probably because the default value from action.yml is not used. The `core.getInput(name)` only reads `INPUT_NAME` env variable. 

To be able to run locally we could
1. Add `INPUT_<config>=<default config url>` env vars to README.dev.md
2. Replace `core.getInput(name)` with `core.getInput(name) || '<default config url>'`
3. Use another method to get defaults from action.yml when running locally
4. Run workflow locally using [act](https://github.com/nektos/act)

What is the best solution?

**Comment by sverhoeven** [comment]
Option 1 could look like
```
INPUT_RULES=https://github.com/NLeSC/tortellini-on-rsd/raw/main/config/rules.kts \
INPUT_CLASSIFICATIONS=https://github.com/NLeSC/tortellini-on-rsd/raw/main/config/license-classifications.yml \
node dist/index.js
```

**Comment by sverhoeven** [comment]
Fixed in #129
